### PR TITLE
TheDocumentFoundation.LibreOffice.LTS version 7.3.5

### PR DESCRIPTION
**Pull request by charlesneimog**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/71338)

### CODE REVIEW

File: manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- xhtml
+- xls
+- xlsb
+- xlsx
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://download.documentfoundation.org/libreoffice/stable/7.3.5/win/x86_64/LibreOffice_7.3.5_Win_x64.msi
```
Comment (ItzLevvie) [inline manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.installer.yaml:39]:
```suggestion
  InstallerUrl: https://downloadarchive.documentfoundation.org/libreoffice/old/7.3.5.2/win/x86_64/LibreOffice_7.3.5.2_Win_x64.msi
```

File: manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  InstallerSwitches:
+    Silent: /qn /norestart
+    SilentWithProgress: /qn /norestart
+  UpgradeBehavior: uninstallPrevious
+  ProductCode: '{44D7B2D9-BAC7-4010-B35E-7A0D54085C66}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://download.documentfoundation.org/libreoffice/stable/7.3.5/win/x86/LibreOffice_7.3.5_Win_x86.msi
```
Comment (ItzLevvie) [inline manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.installer.yaml:50]:
```suggestion
  InstallerUrl: https://downloadarchive.documentfoundation.org/libreoffice/old/7.3.5.2/win/x86/LibreOffice_7.3.5.2_Win_x86.msi
```

File: manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TheDocumentFoundation.LibreOffice.LTS
+PackageVersion: 7.3.5
```
Comment (ItzLevvie) [inline manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.installer.yaml:5]:
```suggestion
PackageVersion: 7.3.5.2
```

File: manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: TheDocumentFoundation.LibreOffice.LTS
+PackageVersion: 7.3.5
```
Comment (ItzLevvie) [inline manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.3.5.2
```

File: manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: TheDocumentFoundation.LibreOffice.LTS
+PackageVersion: 7.3.5
```
Comment (ItzLevvie) [inline manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.3.5/TheDocumentFoundation.LibreOffice.LTS.yaml:5]:
```suggestion
PackageVersion: 7.3.5.2
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/71338#pullrequestreview-1079718365

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to update the version folder from `7.3.5` to `7.3.5.2`

**Comment by charlesneimog** [comment]
Change version from `7.3.5` to `7.3.5.2`

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Inactive PR;

**Pull request closed without merging**